src/components/SetupOpportunity/SetupOpportunity.tsx

```diff
@@ -38,7 +28,12 @@ export const SetupOpportunity: FC<Props> = ({ opportunity }) => {
         opportunity.getOpportunity.websiteListings &&
         opportunity.getOpportunity.websiteListings.items;
 
+    const hasApplications =
+        opportunity.getOpportunity.application &&
+        opportunity.getOpportunity.application.items;
+
     const websiteListings = opportunity.getOpportunity.websiteListings!.items;
+    const applications = opportunity.getOpportunity.application!.items;
```

Comment:
These can be simplified in the same way you did above, by combining the test into the assigning a default, so something like:

```tsx
const websiteListings =         opportunity.getOpportunity.websiteListings &&
opportunity.getOpportunity.websiteListings.items.

Reply:
Fixed in latest push